src/drivers/vercel-kv.ts

```diff
@@ -0,0 +1,43 @@
+import { createClient } from "@vercel/kv";
+import type { RedisConfigNodejs } from "@upstash/redis";
```

Comment:
As i understand, this is a type-only dependency for us. Shall we instead inline the config types?

Reply:
this is a type-level dependency for _us_ but a dependency of `@vercel/kv`. I'll add it back in as an optional peer dep. I think as it is installed anyway, it would be better to use its types.

Reply:
I guess for type support for pnpm users without shamefully-hoist flag they need to install both then right?

Reply:
yes, they would 👍